easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.1.0-fosscuda-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,155 @@
+easyblock = 'PythonBundle'
+
+name = 'TensorFlow'
+version = '2.1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+toolchainopts = {'usempi': True}
```

Comment:
@Flamefire To fix the `scipy` issue on x86_64, you'll need to enable `pic` here too (like we do in `SciPy-bundle`):

```python
toolchainopts = {'pic': True, 'usempi': True}
```

Reply:
Done and tested during the night.